ini/genpeitd.ini

```diff
@@ -1,31 +1,31 @@
-[data]
-name     = Genpei Toumaden
-driver   = System2x
-type     = System86
-filename = gt1_3.6b
-gain     = 0.5
-muterear = 0
-
-[config]
-fm_songtab = 0x4000
-fm_instab = 0x8711
-
-[playlist]
-02 = Theme of Genpei Toumaden
-10 = Game Start
-06 = Small Mode
-21 = Map Mode Intro
-1e = Large Mode
-time = 88
-12 = Theme of Heiankyo
-22 = Theme of Yoshitsune
-time = 72
-1c = Theme of Biwahoushi
-1f = Large Mode (consequent)
-2f = Theme of Benkei
-2a = Bonus Stage
-33 = Yoritomo
-28 = Ending
-18 = Game Over
-31 = Pun Land, Name Entry
-11 = Large Mode Start
+[data]
+name     = Genpei Touma Den

```

Comment:
I think it's not usually written this way.

Reply:
That's the way Namco officially spells it in English.
https://music.apple.com/us/album/genpei-touma-den-original-soundtrack/380065790

Reply:
They're not very consistent with it, though (and they haven't been especially with the iTunes soundtracks)
https://media.vgm.io/albums/80/7208/7208-1209005472.jpg
https://media.vgm.io/albums/30/8903/8903-1473451249.jpg